ibm/service/power/resource_ibm_pi_volumes_attach.go

```diff
@@ -0,0 +1,99 @@
+// Copyright IBM Corp. 2024 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package power
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	st "github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/power-go-client/power/models"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func ResourceIBMPIVolumesAttach() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMPiVolumesAttachCreate,
+		ReadContext:   resourceIBMPiVolumesAttachRead,
+		DeleteContext: resourceIBMPiVolumesAttachDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			// Arguments
+			Arg_CloudInstanceID: {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The GUID of the service instance associated with an account.",
+			},
+			Arg_InstanceID: {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The unique identifier of the instance.",
```

Comment:
For required properties I've been adding ValidateFunc: validation.NoZeroValues to try to avoid accepting empty values when it doesn't work? Does this make sense here?